webpack.config.js

```diff
@@ -25,10 +29,23 @@ if (process.env.NODE_ENV === 'production') {
 }
 
 module.exports = {
-  entry: path.join(JS_ROOT, 'entry.js'),
+  entry: {
+    huxley: path.join(JS_ROOT, 'entry.js'),
+    vendor: [
+      'classnames',
+      'flux',
+      'jquery',
+      'jquery-ui',
+      'js-cookie',
+      'react',
+      'react-dom',
+      'react-modal',
+      'react-router',
+    ],
+  },
   output: {
     path: STATIC_ROOT,
-    filename: 'bundle.js',
+    filename: '[name].js',
```

Comment:
Just a bit confused about how webpack works, but is this referring to vendor or huxley?